app/views/helpers/index/normal/entry_bottom.phtml

```diff
@@ -87,6 +87,7 @@
 						}
 						$share_options['link'] = $link;
 						$share_options['title'] = $title;
+						$share_options['content'] = $this->entry->content();
```

Comment:
Hum, we have a risk for duplicated the article content many times here, right?
I think this should be replaced by a dynamic menu, either server side or client side

Reply:
I am not sure I follow your reasoning. Could you explain it?

Reply:
Yes: I am worried about the output size. If I am not mistaken, the HTML output will contain up to several times the content of each article, considerably increasing the size of the page

Reply:
Ok, I understand all the concerns. I think it's better if we keep it the way it is. I am closing this PR.

Reply:
Ok. Do you see any problem with a dynamic menu (either client side or server side)?

Reply:
I think that if one want the content, he will copy and paste it as I am doing at the moment. I don't think that spending time on that is necessary.

Reply:
Ok